examples/gaia_agent/README.md

```diff
@@ -18,10 +18,24 @@ The agent is free to choose which thoughts and actions to use to satisfy the cur
 Additionally, the Gaia agent implements the initial planning step, which produces a «plan» in the form of a sequence of free-form descriptions of the actions that should be taken to solve the task.
 
 ## Quickstart
+Perform all the following steps from the top folder of the repo.
+First, you need to install the dependencies for file converters and make folders:
+```bash
+pip install -r requirements.converters.txt
```

Comment:
Not sure if this command actually installed anything for me. And even after running it, there's a bunch of errors in my logs (will share with you).